flink-fault-tolerance/README.md

```diff
@@ -0,0 +1,77 @@
+# Overview
+
+This example shows how to make the Feathub job fault-tolerant when running with 
+FlinkProcessor.
+
+# Prerequisites
+
+Prerequisites for running this example:
+- Unix-like operating system (e.g. Linux, Mac OS X)
+- Python 3.7
+- Docker Compose >= 20.10
+
+# Step-By-Step Instructions
+
+Please execute the following commands under the `flink-fault-tolerance`
+folder to run this example.
+
+1. Install FeatHub pip package with FlinkProcessor dependencies.
+
+   ```bash
+   $ python -m pip install --upgrade "feathub-nightly[flink]"
+   ```
+
+2. Start the Flink and the Kafka cluster.
+
+   ```bash
+   $ docker-compose up -d
+   ```
+
+   After the Flink and the Kafka clusters have started, you should be able to
+   navigate to the web UI at [localhost:8081](http://localhost:8081) to view the
+   Flink dashboard.
+
+3. Produce events into the Kafka topic..
+
+   ```bash
+   $ python initialize_kafka_topic.py
```

Comment:
Given that this step uses DataGenSource, is the output deterministic in such a way that user can verify that the FeatHub job's state is preserved after failover?

If yes, it might be useful to explain how to verify this in the README.

If no, then maybe we can re-use the same steps used in `run_and_verify.sh`, such that the output is deterministic and can demonstrates that the output result is not affected by the fail over.

Reply:
You are right. We should re-use the steps in `run_and_verify.sh`.